.github/actions/e2e-tests/action.yml

```diff
@@ -43,6 +43,9 @@ runs:
           ENABLE_INFLUXDB_FEATURES: ${{ inputs.enable_influxdb_features }}
 
         run: |
+          wget -q https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_96.0.4664.110-1_amd64.deb
```

Comment:
I'm sure this was only recently removed by @dabeeeenster, how come it's being re-added?

Reply:
I'll revert this, I bet the concurrency will sort the freezes